packages/core/useBluetooth/index.ts

```diff
@@ -87,23 +87,27 @@ export function useBluetooth(options?: UseBluetoothOptions): UseBluetoothReturn
     }
   }
 
-  const server = ref<undefined | BluetoothRemoteGATTServer>()
+  const server = shallowRef<undefined | BluetoothRemoteGATTServer>()
+  const isConnected = ref(false)
 
-  const isConnected = computed((): boolean => {
-    return server.value?.connected || false
-  })
+  function reset() {
+    isConnected.value = false
+    device.value = undefined
+    server.value = undefined
+  }
 
   async function connectToBluetoothGATTServer() {
     // Reset any errors we currently have:
     error.value = null
 
     if (device.value && device.value.gatt) {
-      // Add callback to gattserverdisconnected event:
-      device.value.addEventListener('gattserverdisconnected', () => {})
+      // Add reset fn to gattserverdisconnected event:
+      device.value.addEventListener('gattserverdisconnected', reset)
```

Comment:
This should be replaced with ``useEventListener`` so it's disposed accordingly.

Reply:
Nice spotting! We could do that in another PR